src/Board.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by lbregula on 11/29/16.
+ */
+public class Board {
+    private int length;
```

Comment:
As noted in class the length and the width are kind of strange in this context.  Things that might be more useful would be `private Property[] properties` etc.